### PR TITLE
Feature/paragraphs

### PR DESCRIPTION
**Pull request by PablitoGS**
I' ve edited markdown, please review @larnedo @nguasch

### CODE REVIEW

File: src/base/_paragraphs.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+p {
+  font-size: $fz-base;
+  line-height: $lh-base-body;
+  margin-bottom: $m-base;
+  word-wrap: break-word;
+
+  &:last-child,
+  &:only-child {
+    margin-bottom: 0;
+  }
```
Comment (nguasch) [inline src/base/_paragraphs.scss]: I fixed this indentation

Comment (PablitoGS) [inline src/base/_paragraphs.scss]: :+1:

**Comment by nguasch** [comment]
:+1:

**Comment by larnedo** [comment]
I'm not sure if dist directory should be published? @PablitoGS @nguasch

**Comment by nguasch** [comment]
@larnedo We think that if it's an open source project, you should be able to download the compiled styles ready to use, without having to do anything else

**Comment by larnedo** [comment]
@nguasch This is not what are doing other open source frameworks lie SUITCSS or inuitcss, but in that case I think that we should included the dist CSS's without minimizing. What doy you think? @nguasch @PablitoGS @nucliweb

**Comment by larnedo** [comment]
@PablitoGS @nguasch We can talk about what I comment out of this conversation.

Go ahead with the changes! :+1:

**Pull request merged**